cypress/e2e/Portfolio/Load&SaveEditor.cy.js

```diff
@@ -0,0 +1,56 @@
+// import {signIn} from '../DoenetSignin/DoenetSignin.cy';
+
+const { cesc2 } = require("../../../src/_utils/url");
+
+describe("doenetEditor test", function () {
```

Comment:
These test descriptions should be updated, also note from @dqnykamp , he forgot to do this in ShareActivities, so we should also change that one (you didn't edit close enough to that line on the file for github to let me make a comment there)

Reply:
This file has changed.  I'm not sure if this is already fixed.